src/recipe/parser/snapshots/rattler_build__recipe__parser__package__tests__missing_fields.snap

```diff
@@ -0,0 +1,14 @@
+---
+source: src/recipe/parser/package.rs
+expression: err
+---
+  × Parsing: Missing field `version`
+   ╭─[2:1]
+ 2 │         package:
+ 3 │             name: test
```

Comment:
Is this the correct position for the span?
I am wondering if underlining `package` would be better?

Reply:
I think underlining `package` would be better as well. But by making the conversion generic over a trait (as it is right now) we lose the ability to reference.

One possible option is to change the trait definition to receive a `parent_span` as a parameter, to reference it if needed. What do you think?